src/widgets/plots/pie_plot.py

```diff
@@ -6,7 +7,20 @@ def __init__(self, data):
         super().__init__(data)
 
     def plot(self):
-        ax = self.canvas.figure.subplots()
-        counts = self.data.value_counts().to_dict()
-        ax.pie(counts.values(), labels=counts.keys())
-        return self.canvas
+        try:
+            ax = self.canvas.figure.subplots()
+            counts = self.data.value_counts().to_dict()
+            data_size = self.data.size
+            labels = [k if counts[k]/data_size > self.min_pie_plot_label_ratio else '' for k in counts.keys()]
```

Comment:
Maybe plot few big pieces and set label for them and other values merge to one piece in circle and set label 'other'. This one piece may be in one color or divide like now.